migrator/migrator.go

```diff
@@ -179,22 +179,25 @@ func (p *PouchMigrator) PreMigrate(ctx context.Context, takeOverContainer bool)
 			return err
 		}
 
+		var imageName string
 		// meta.Image maybe a digest, we need image name.
 		image, err := p.dockerd.ImageInspect(meta.Image)
-		if err != nil {
+		if err != nil && strings.Contains(err.Error(), "No such image") {
```

Comment:
please do not make an error string to compare with an case-sensitive sub string. Otherwise, that would bring some unstable factor.

Please use `strings.Contains(string.ToLower(err.Error()), "no such image")`